core/controllers/editor_test.py

```diff
@@ -526,19 +529,21 @@ def test_exploration_download_handler_for_default_exploration(self):
         # Check downloaded zip file.
         filename = 'oppia-ThetitleforZIPdownloadhandlertest!-v2.zip'
         self.assertEqual(response.headers['Content-Disposition'],
-                         'attachment; filename=%s' % str(filename))
-        zf_saved = zipfile.ZipFile(StringIO.StringIO(response.body))
+                         'attachment; filename=%s' % filename)
+        zf_saved = zipfile.ZipFile(
+            python_utils.string_io(buffer_value=response.body))
         self.assertEqual(
             zf_saved.namelist(),
             ['The title for ZIP download handler test!.yaml'])
 
         # Load golden zip file.
-        with open(os.path.join(
+        with python_utils.open_file(os.path.join(
```

Comment:
Try breaking after the first '(' instead. Maybe use a intermediate variable for os.path.join(..., ...).

Reply:
Done

Reply:
Sorry it was actually done for constants. Done for this file too!